### ISSUE TITLE
Uncaught ReferenceError: device is not defined

### ISSUE THREAD

**Issue by pmn4** [driftyco/ng-cordova#75]
##### tl;dr: would you mind adding an `enabled` method/flag to each module?

_Use Case:_
I run my apps using [ripple](http://emulate.phonegap.com/) (directly in my desktop browser) while developing, so plugins are typically unavailable.  Let's say I want to log device information on startup (e.g, `$log.log($cordovaDevice.getDevice())`), I would get a ReferenceError in the browser (device is not defined).

I would like to do a quick check that the plugin is enabled.  something like:

``` javascript
$log.log($cordovaDevice.enabled() ? $cordovaDevice.getDevice() : "device n/a");
```

Thanks for this!  (and ionic!)

---

alternatively, could you [return `window.device`](https://github.com/driftyco/ng-cordova/blob/master/src/plugins/device.js#L7) from `getDevice()` for the times when `device` is not defined?
